README.md

```diff
@@ -36,12 +37,24 @@ This extension contributes the following settings:
   - `visible`: check only visible files
   - `open`: check all open files
   - `project`: check the entire project's files
+* `lean.input.leader`: character to type to trigger Unicode input mode (`\` by default)
+* `lean.input.languages`: allows the Unicode input functionality to be used in other languages
+
+It also contributes the following commands, which can be bound to keys if desired:
+
+* `lean.input.convert`: converts the current Unicode abbreviation (bound to <kbd>tab</kbd> by default)
+* `lean.infoView.displayGoal`: show the tactic state and any messages (e.g. info, warning, error) at the current position in the info view window (bound to <kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>enter</kbd> by default)
+* `lean.infoView.displayList`: show all messages for the current file from Lean in the info view window (bound to <kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>alt</kbd>+<kbd>enter</kbd> by default)
+* `lean.infoView.copyToComment`: copy the current contents of the info view into a new comment on the next line
+* `lean.infoView.toggleUpdating`: pause / continue live updates of the info view
+* `lean.roiMode.select`: select the region of interest (files to be checked by the Lean server)
+* `lean.batchExecute`: execute the current file using Lean (bound to <kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>r</kbd> by default)
 
 ### Other potentially helpful settings
 
-* Fonts with good unicode support (on Linux): `"editor.fontFamily": "Source Code Pro Medium, DejaVu Sans Mono"`
```

Comment:
Please keep the configuration example here.  For this configuration of fonts to work properly, they need be specified both and in this order (so that characters that are not available in Source Code Pro are rendered using DejaVu Sans Mono).